backend/node_app/config/constants.js

```diff
@@ -40,7 +40,7 @@ module.exports = Object.freeze({
 		}
 	},
 	GAMECHANGER_BACKEND_BASE_URL: `http://${process.env.GAMECHANGER_BACKEND_HOST}:8990`,
-	GAMECHANGER_BACKEND_EDA_URL: `http://${process.env.GAMECHANGER_BACKEND_EDA_HOST}:8990`,
+	GAMECHANGER_BACKEND_EDA_URL: `http://${process.env.EDA_DATA_HOST}:8990`,
```

Comment:
Can you add the EDA_DATA_HOST to the env.template file as well as to the confluence page that has the .env file that way we know to include the right stuff into the deployments?